server/models/helpers/NotificationHelper.ts

```diff
@@ -164,12 +164,13 @@ export default class NotificationHelper {
       const collectionIds = await recipient.collectionIds();
 
       // Check the recipient has access to the collection this document is in. Just
-      // because they are subscribed doesn't meant they "still have access to read
+      // because they are subscribed doesn't mean they still have access to read
       // the document.
       if (
         recipient.email &&
         !recipient.isSuspended &&
-        collectionIds.includes(document.collectionId)
+        (!document.collectionId ||
+          collectionIds.includes(document.collectionId))
```

Comment:
```suggestion
        document.collectionId && collectionIds.includes(document.collectionId))
```

Reply:
Should we limit it to the creator then in case of draft without collection?

Reply:
For now it's not possible to get a notification in a draft without collection, but in the future this will need updating when we have individually shared documents _soon_